zrml/prediction-markets/src/lib.rs

```diff
@@ -1167,11 +1172,49 @@ mod pallet {
         pub(crate) fn internal_resolve(market_id: &T::MarketId) -> Result<Weight, DispatchError> {
             let market = Self::market_by_id(market_id)?;
             let report = market.report.clone().ok_or_else(|| NO_REPORT)?;
-            let mut total_accounts = 0u32;
-            let mut total_asset_accounts = 0u32;
-            let mut total_categories = 0u32;
+            let total_accounts = 0u32;
+            let total_asset_accounts = 0u32;
+            let total_categories = 0u32;
```

Comment:
@sea212 Liquidity removal must happen before market resolution so these parameters should be removed or transferred to other entry-point. Any suggestions?

Reply:
We have to transfer this to the function that removes liquidity. As I see it currently, the function `report` does it now. We have to reflect the changes in the benchmarks:
1) Think about which logical paths the functions `report` and `internal_resolve` take now on from which variables they depend
2) Update benchmarks
3) Update `report` and `internal_resolve` to return overcharged weight
4) Benchmark new weights and include into `weight.rs`

I can handle the 4th step after you're done, so all the weights were consistently calculated on the same hardware. We'll implement an automatic solution for that soon.

Reply:
Sorry @sea212, I was a bit confused about what should be done and these parameters transfer shouldn't be necessary anymore